src/App.css

```
@@ -141,6 +141,20 @@
   margin: 0.5rem auto 1rem auto;
 }
 
+.Build-System__list {
+  display: flex;
+  justify-content: center;
+  gap: 20px;
+
+  max-width: 800px;
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)